src/components/React/VolumeRenderer.js

```diff
@@ -77,7 +77,7 @@ export default connect(mapStateToProps)(
                   id="volumeCube"
                   class="clickableMesh"
                   myloader={{
-                    volumeData: state.path,
+                    volumeData: state.options,
```

Comment:
Are you using `volumeData` anymore? If not we should probably delete it. `state.options` has been renamed.
```suggestion
```

Reply:
Removed !